component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1036,6 +1076,23 @@ protected boolean retryAuthenticationEnabled() {
         valuesToBeMasked.setDisplayOrder(6);
         configProperties.add(valuesToBeMasked);
 
+        Property mobileNumberRegex = new Property();
+        mobileNumberRegex.setName(SMSOTPConstants.MOBILE_NUMBER_REGEX);
+        mobileNumberRegex.setDisplayName("Mobile number Regex Pattern");
```

Comment:
use consistent format for displayname. e.g. start every word with capital.

Mobile Number Regex Pattern